src/selectedtests/app/controllers/project_test_mappings_controller.py

```diff
@@ -0,0 +1,81 @@
+"""Controller for the health endpoints."""
+import json
+
+from flask import jsonify, request
+from flask_restplus import abort, Api, fields, Resource
+from evergreen.api import EvergreenApi
+
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.app.test_mapping_work_item import TestMappingWorkItem
+from selectedtests.evergreen_helper import get_evg_project
+
+MONGO_WRAPPER = None
+EVG_API = None
+
+
+def add_project_test_mappings_endpoints(api: Api, mongo: MongoWrapper, evg_api: EvergreenApi):
+    """
+    Add to the given app instance the test mapping jobs endpoints of the service.
+
+    :param api: An instance of a Flask Restplus Api that wraps a Flask instance
+    :param mongo: Mongo Wrapper instance
+    :param evg_api: An instance of the evg_api client
+    """
+    global MONGO_WRAPPER
```

Comment:
I'm not a fan of using global variables like this. It makes the code really difficult to refactor (I've recently had to remove a bunch of these from build-baron-tools to be able to extend it). Plus, I don't think you need global variables in this case, you should just be able to use the function parameters.